modes/lisp-mode/lisp-mode.lisp

```diff
@@ -319,6 +319,16 @@
     (start-lisp-repl)
     (buffer-end (buffer-point repl-buffer))))
 
+(define-command lisp-current-directory () ()
+  (message "Current directory: ~a" *default-pathname-defaults*) )
+
+(define-command lisp-set-directory (&key directory) ()
+  (unless directory
+    (setf directory
+           (prompt-for-directory "New directory: " :directory (buffer-directory))))
+  (swank:set-default-directory 
+   (swank-backend:filename-to-pathname directory)))
```

Comment:
Perhaps this only works for self-connection?
If you are connected to another process, you may want to use lisp-eval or something similar.
https://github.com/lem-project/lem/blob/main/modes/lisp-mode/lisp-mode.lisp#LL259C5-L259C5

Reply:
Oh, you are right indeed! 

Thanks for the catch :+1: , I update the code to reflect it